src/ray/raylet/node_manager.cc

```diff
@@ -2937,57 +2937,38 @@ MemoryUsageRefreshCallback NodeManager::CreateMemoryUsageRefreshCallback() {
           RAY_LOG_EVERY_MS(WARNING, 5000) << "Worker killer did not select a worker to "
                                              "kill even though memory usage is high.";
         } else {
+          RAY_LOG(INFO)
+              << "Killing worker with task "
+              << worker_to_kill->GetAssignedTask().GetTaskSpecification().DebugString();
           high_memory_eviction_target_ = worker_to_kill;
 
-          float usage_fraction =
-              static_cast<float>(system_memory.used_bytes) / system_memory.total_bytes;
-          std::string used_bytes_gb = FormatFloat(
-              static_cast<float>(system_memory.used_bytes) / 1024 / 1024 / 1024, 2);
-          std::string total_bytes_gb = FormatFloat(
-              static_cast<float>(system_memory.total_bytes) / 1024 / 1024 / 1024, 2);
-          std::stringstream id_ss;
-          if (worker_to_kill->GetActorId().IsNil()) {
-            id_ss << "task ID " << worker_to_kill->GetAssignedTaskId();
-          } else {
-            id_ss << "actor ID " << worker_to_kill->GetActorId();
-          }
+          /// TODO: (clarng) expose these strings in the frontend python error as well.
+          static std::string oom_kill_title =
+              "Task was killed due to the node running low on memory. ";
+          std::string oom_kill_details = this->CreateOomKillMessageDetails(
+              worker_to_kill, this->self_node_id_, system_memory, usage_threshold);
+          std::string oom_kill_suggestions =
+              this->CreateOomKillMessageSuggestions(worker_to_kill);
+
+          RAY_LOG_EVERY_MS(ERROR, 10000) << oom_kill_title << "\n\n"
```

Comment:
why don't we just change it to INFO? Since the error is already printed, this seems to be a duplicated error message, and not very useful at the drvier